lint-tasks.yml

```diff
@@ -4,7 +4,12 @@ vars:
   G_LINT_VENV_DIR: "{{.BUILD_DIR}}/lint-venv"
   G_LINT_VENV_CHECKSUM_FILE: "{{.BUILD_DIR}}/lint#venv.md5"
   # Linter target dirs
-  G_CPP_LINT_DIRS: ["{{.CLP_FFI_PY_CPP_SRC_DIR}}"]
+  G_CPP_LINT_DIRS:
+    # TODO: before all clang-tidy warnings are resolved, we should maintain a list of files that
+    # doesn't have clang-tidy warnings, so that clang-tidy can be executed in the limited scope
+    # without failing existing workflows.
```

Comment:
```suggestion
    # TODO: Before all clang-tidy violations are resolved, we should only run clang-tidy on the
    # files whose violations we've fixed, both to ensure they remain free of violations and so that
    # the workflow doesn't fail due to violations in other files.
```